libs/aws/langchain_aws/utils.py

```diff
@@ -1,21 +1,51 @@
 import re
+import sys
 from typing import Any, List
 
+from packaging import version
+
 
 def enforce_stop_tokens(text: str, stop: List[str]) -> str:
     """Cut off the text as soon as any stop words occur."""
     return re.split("|".join(stop), text, maxsplit=1)[0]
 
 
-def _get_anthropic_client() -> Any:
+def check_anthropic_tokens_dependencies() -> List[str]:
```

Comment:
nit: Would anthropic_tokens_supported be a better name here?

Reply:
Updated this 👍